addons/web_editor/__manifest__.py

```diff
@@ -108,6 +108,10 @@
             'web_editor/static/src/js/wysiwyg/dialog.js',
             'web_editor/static/src/js/frontend/loader.js',
         ],
+        'web.report_assets_common': [
+            'web_editor/static/src/scss/bootstrap_overridden.scss',
```

Comment:
if done, maybe we should probably remove this from here and add it to `web._assets_helpers` (that's in backend, common and frontend assets as far as I can tell) but note sure about how to get the correct ordering in this case if removed from web._assets_backend_helpers and web._assets_frontend_helpers